### PR TITLE
VR > Loader

### PR DESCRIPTION
**Pull request by juanfra**
🎫 https://central.tri.be/issues/135105
🎥 https://www.loom.com/share/e6518b7c3ef64688b4a86842aa8a8f66

### CODE REVIEW

File: src/views/v2/components/loader.php [hunk 1]
```diff
@@ -11,14 +11,17 @@
  *
  * @version TBD
  *
+
```
Comment (paulmskim) [inline src/views/v2/components/loader.php]: extra line?

File: src/views/v2/components/loader.php [hunk 2]
```diff
@@ -11,14 +11,17 @@
...
 ?>
 <div
 	class="tribe-events-view-loader tribe-common-a11y-hidden"
 	role="alert"
 	aria-live="assertive"
 >
-	<div class="tribe-events-view-loader__spinner">
-		<?php echo esc_html( $text ); ?>
-	</div>
+		<div class="tribe-events-view-loader__dots tribe-common-c-loader">
```
Comment (paulmskim) [inline src/views/v2/components/loader.php]: spacing is off here and below

Comment (paulmskim) [reply src/views/v2/components/loader.php]: btw nice use of classes here :+1: really liking it.

Comment (juanfra) [reply src/views/v2/components/loader.php]: Thank you!

**Pull request merged**